### ISSUE TITLE
Problem packing the dohnadohnaPact.afa file

### ISSUE THREAD

**Issue by SUNCRIS** [nunuhara/alice-tools#26]
Hi i have been been trying to modify the dohnadohnaPact.afa file, had no problems extracting but when i try to pack them back with the manifest that is posted in https://github.com/nunuhara/alice-tools/issues/16:

#BATCHPACK
dohnadohnaPact.afa
x,x,pactex,pactex

This error appears: ERROR: line 2: "pactex" is not a directory

I have the x folder and the manifest file in same folder.

I tried writing other manifests and the ones that worked without errors make the dohnadohnaPact.afa file have 10 times the original size and dont work ingame...

**Comment by nunuhara** [comment]
It seems I missed a step in the instructions. Before running the `alice ar pack dohnadohnaPact.manifest` command, you need to create a directory named `pactex` in the same directory as the manifest file. Then it should work.

I'll look into having alice-tools automatically create this directory in the next release.

**Comment by SUNCRIS** [comment]
Ok first of all, now im able to make the dohnadohnaPact.afa but the file size is 1089kb, still weird because original .afa is 615kb and for example Eraea's .afa(posted here https://www.mediafire.com/file/ja7nyx4b9f5fgcs/TranslateVerERAEA-0.41-fixed.7z/file) is 762kb.

So when i tried the game, the options button was not working and when you enter in a battle the game just crashes.

I think it is because the battle(Scene\60_Battle\Main\Stage\Background) and options files in (Scene\90_System\Config) are the only ones in japanese letters.

Then i tried changing my system language to japanese, i packed the .afa (the file size is now 1085kb without changing anything?) and all worked perfect ingame.

Another test i made was extract AND pack the .afa with my system language in japanese and this time it didn't worked ingame...

So basically, i have to extract the .afa with my normal system language and then change the language to be able to pack a working .afa...

Idk if having to change your language is something expected and im just dumb or what XD, but i just wanted u to know.

**Comment by Cerzs** [comment]
I'm having the same problem. The new file is larger than the original and does not work.

**Comment by crisdiaz-github** [comment]
> I'm having the same problem. The new file is larger than the original and does not work. I have a lot of odd names like this: Scene/60_Battle/Main/Stage/Background/**a?a??a?.pactex.x**

Idk if u are having the same error, but i solved it manually changing my system locale to japanese for being able to use the pack command.

So basically extract and edit the files normally, change to japanese locale, restart pc and finally pack.

**Comment by SUNCRIS** [comment]
> I'm having the same problem. The new file is larger than the original and does not work. I have a lot of odd names like this: Scene/60_Battle/Main/Stage/Background/**a?a??a?.pactex.x**

Idk if u are having the same error, but i solved it manually changing my system locale to japanese for being able to use the pack command.

So basically extract and edit the files normally, change to japanese locale, restart pc and finally pack.

**Comment by Cerzs** [comment]
@SUNCRIS I unpacked the .afa without japanese locale, then tried to pack using jp locale and got this error:

`ERROR: stat("Scene/60_Battle/Main/Stage/Background/a?a??a?.pactex.x"): No such file or directory`

**Comment by SUNCRIS** [comment]
[dohnadohnaPact.zip](https://github.com/nunuhara/alice-tools/files/7449641/dohnadohnaPact.zip)
Weird, try with this one.

**Comment by Cerzs** [comment]
Ah, I'm using the English version that was released a couple of days ago, maybe it's different :/

**Comment by SUNCRIS** [comment]
Yeah i used the japanese version.

I was making a translation back before the anouncement XD https://f95zone.to/threads/translation-request-alicesoft-dohna-dohna-isshou-ni-warui-koto-wo-shiyou.67577/post-6499463 u have all the files there, maybe it helps you.

**Comment by Cerzs** [comment]
@nunuhara Do you have any idea? I don't know if this would be useful, but [here's the English afa](https://cdn.discordapp.com/attachments/687043665289412705/904600271479525406/dohnadohnaPact.afa).

**Comment by nunuhara** [comment]
This should now be fixed in the latest release (0.10.1)

**Comment by Cerzs** [comment]
@nunuhara Got this error (Eng version):
![imagen](https://user-images.githubusercontent.com/44489935/140256826-5dba1bd0-d840-4f38-9891-da601208eeac.png)
